data/Republic Navy Intro.txt

```diff
@@ -0,0 +1,265 @@
+			choice
+				`	"Consider it done."`
+				`	"I'm sorry, but I can't deal with problems like that."`
+					decline
+			`	Captain Shah gives a small, relieved nod. "The ship was last spotted in Tias. We believe it is still there."`
+				accept
+	npc disable
+		personality heroic staying nemesis target
+		government Bounty
+		system "Tais"
+		ship "Falcon (Heavy)" "Gediminas"
+		dialog `The <npc> has been disabled. You can claim the bounty payment by returning to <destination>.`
+	on accept
+		log "Hunting a pirate ship called the "Gediminas", who has captured multiple merchant captains. The ship was last sighted in the Tais system."
+		
+	on complete
+		"assisted Republic" ++
+		payment 250000
+		dialog `Captain Shah contacts you soon after you land. "The hostages were recovered safely." she says, with a smile. "The Republic owes you a debt of gratitude. If you are interested in more tangible benefits, <payment> have been transferred to your account, and I have informed my superiors that you can be counted on."`
+		log "Captain Shah says she has put a good word in for me with her superiors."
+
+		
+mission "Navy Scouting 1"
+	name "Hyperspace Relay Testing"
+	description "Fly through the <waypoints> systems, then report to <destination> so the Navy can analyze your logs network connectivity."
+	waypoint "Hassaleh"
+	waypoint "Sumar"
+	waypoint "Cardax"
+	waypoint "Volax"
+	source
+		government "Republic" 
+		attributes "paradise"
+	destination "Glaze"
+	to offer
+		Has "event: war begins"
+		not "chosen sides"
+		random < 60
+
+	on offer
+		conversation
+			`As you wander the spaceport you overhear a woman in a Navy uniform telling a bored looking merchant about satellight dishes in the empty systems to the north. `
+			`	"That sounds interesting." The merchant tells her, "But I've got a cargo bay full of medical supplies headed the other direction." "Also an important mission." She replies. "I wish you safe travels."`
+			choice
+				`	(Ask her what's going on)`
+				`	(Walk away)`
+					decline
+			`	"Are you needing someone to go retrieve a satellight?" You ask her, interested, as the other merchant captain walks away. "Not quite." She replies brightly. "We've been experiencing some lag between Farpoint and Maelstrom. I'd need you to fly through, then let me look at your logs so I can figure out what the trouble is."`
+			choice
+				`	"Sounds simple enough."`
+				`	"Sorry, I won't be able to help you either."`
+					decline
+			`	"Excellent." She smiles. "I'm Captain Jane Mulgrew, by the way. Thank you for your help."`
+				accept
+
+	on accept
+		log "Volunteered to help track down the cause of some lag for the Republic."
+	
+	on complete
+		payment 30000
+		dialog
+			`As you land on <destination> the woman approaches you again. "Thank you on behalf of not only the Republic, but everyone who uses my network. You have earned your payment of <payment>"`
```

Comment:
Do we want to name the person here seeing as the name is already know.
And is the network hers or the republics.

Reply:
I see the network as being a series of essentially internet relays. The Republic uses, deploys, and maintains some of the relays, but others are handled by private organizations - such as the Syndicate, utilities companies, and beyond. 

This specific reach of space is maintained by the Republic, but is a public fund